minifi/minifi-nar-bundles/minifi-framework-bundle/minifi-framework/minifi-runtime/src/main/java/org/apache/nifi/minifi/MiNiFi.java

```diff
@@ -45,7 +49,8 @@
 
 public class MiNiFi {
 
-    private static final Logger logger = LoggerFactory.getLogger(MiNiFi.class);
+    private static final Logger LOGGER = LoggerFactory.getLogger(MiNiFi.class);
```

Comment:
Although the repository is not completely consistent, recommend leaving this lowercase name of `logger` unchanged.